lib/sunspot_test.rb

```diff
@@ -23,13 +23,18 @@ def server
 
     def start_sunspot_server
       unless solr_running?
-        pid = fork do
+        # TODO remove forking (only call server.start) if sunspot_solr > 2.2.7
+        fork do
           STDERR.reopen("/dev/null")
-          STDOUT.reopen("/dev/null")
-          server.run
+          server.start
```

Comment:
Why `start` rather than `run`?

Reply:
Yes sunspot manages all the pid file handling and the forking when we use `start` and `stop`.
The only downside I see is that you can only start one test sunspot server because they use all the same pid file. But I don't know if it is possible to run multiple instances at all.

We could also do something like this:
https://github.com/justinko/sunspot-rails-tester/commit/680a5d985afd336190294aad68caa80d42349074